source/MaterialXTest/ShaderValid.cpp

```diff
@@ -24,11 +24,20 @@ namespace mx = MaterialX;
 #include <MaterialXView/Handlers/TinyEXRImageHandler.h>
 #include <MaterialXView/Handlers/LightHandler.h>
 
+// #define LOG_TO_FILE
```

Comment:
Thanks for the cleanup here. Maybe we should always log to file by default.

Reply:
OK I will change the default.